themes/default/content/automation/_index.md

```diff
@@ -6,4 +6,26 @@ meta_title: Automation API
 meta_desc: |
     Use infrastructure as code directly from your favorite language to
     create custom tools, portals, and frameworks for any cloud architecture.
+
+customer_logos:
```

Comment:
I don't think we need the logo reel on this page since it already appears on the main page and we've got customer examples in the section above this